src/switch/switch.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+  Switch as NativeSwitch,
+  StyleProp,
+  ViewStyle,
+  ViewProps,
+  Platform,
+} from 'react-native';
+import { withTheme } from '../config';
+import { Theme } from '../config/theme';
+
+export type SwitchProps = {
```

Comment:
I don't understand why you rewrote the props here. Shouldn't this just be `SwitchProps` from React-Native?

Reply:
Yes it can be, but I thought of using the types here, I am working on your review and will change it.